src/BlazorWebView/src/Maui/Android/WebKitWebViewClient.cs

```diff
@@ -177,16 +182,16 @@ protected override void Dispose(bool disposing)
 
 		private class JavaScriptValueCallback : Java.Lang.Object, IValueCallback
 		{
-			private readonly Action _callback;
+			private readonly Action<Java.Lang.Object?> _callback;
 
-			public JavaScriptValueCallback(Action callback!!)
+			public JavaScriptValueCallback(Action<Java.Lang.Object?> callback!!)
```

Comment:
This is inline with the API:

- https://developer.android.com/reference/android/webkit/WebView?hl=en
- https://developer.android.com/reference/android/webkit/ValueCallback
- https://docs.microsoft.com/en-us/dotnet/api/?view=xamarin-android-sdk-12
- https://docs.microsoft.com/en-us/dotnet/api/android.webkit.ivaluecallback?view=xamarin-android-sdk-12